src/ingest_validation_tools/table-schemas/level-2/lcms.yaml

```diff
@@ -10,9 +10,9 @@ fields:
     name: assay_type
     constraints:
       enum:
-        - LC-MS (metabolomics)
-        - LC-MS/MS (label-free proteomics)
-        - MS (shotgun lipidomics)
+        - TMT LC-MS
+        - Targeted Shotgun / Flow-injection LC-MS
+        - Untargeted LC-MS
```

Comment:
Labeled LC-MS/MS
Label-free LC-MS/MS
Label-free MS/MS

Reply:
Again, not the current version.